__tests__/app.test.js

```diff
@@ -44,3 +41,43 @@ describe("TASK 3.5 --- GET - /api", () => {
       });
   });
 });
+describe.only("TASK 4 --- GET - /api/articles/:article_id", () => {
+  it("GET - status: 200 - responds with JSON object containing correct article contents", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        console.log(response.body)
+        expect(response.body.result.article_id).toBe(1)
+        expect(response.body.result.title).toBe("Living in the shadow of a great man");
+        expect(response.body.result.topic).toBe("mitch");
+        expect(response.body.result.author).toBe("butter_bridge");
+        expect(response.body.result.body).toBe("I find this existence challenging");
+        expect(response.body.result.created_at).toBe("2020-07-09T19:11:00.000Z");
+        expect(response.body.result.votes).toBe(100);
+        expect(response.body.result.article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+      });
+  });
+  it('GET - status: 400 - responds with error message explaining it was a bad request', () => {
+    return request(app)
+    .get(`/api/articles/nonsense`)
+    .then((response) => {
+      console.log(response);
+      expect(response.statusCode).toBe(400)
+      expect(JSON.parse(response.text)).toEqual({ msg: "Bad Request" })
```

Comment:
You shouldn't need to parse the error response. What you're sending back should be easily grabbed on the response body.